strings_lab.js

```diff
@@ -0,0 +1,220 @@
+// // 1. `console.log` the first character of a string.
+let string = "Hello"
+console.log(string[0]);
+//
+//
+// // 2. `console.log` the length of a string.
+ let string = "Hello"
+console.log(string.length);
+
+// // 3. `console.log` the last character of any string.
+ let string = "Hello"
+console.log(string[string.length - 1]);
+
+
+// 4. Create a code block that takes a single string variable, and returns a
+// copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+
+let string = "hello"
+
+let newString = string.slice(0,(string.length - 1)) + string[string.length - 1].toUpperCase();
+
+ console.log(newString);
+
+
+
+// 5. Create a drEvil code block that will take a single number variable, and
+// log the '<variableAmount> dollars', your code block should add '(pinky)' at
+// the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+
+let amount = 1000000
+if(amount === 1000000){
+  console.log(amount + " dollars (pinky)")
+}
+else{
```

Comment:
Practice on improving indentation and line breaking for your code. It will make your code more legible that way. Come see me if you have any questions.